src/site/content/en/blog/economic-times-cwv/index.md

```diff
@@ -0,0 +1,163 @@
+</figure>
+
+## What is First Input Delay (FID) and how did we improve it?
+
+[First Input Delay](/fid/) is the metric that tracks a website's responsiveness to user input. The primary cause of a poor FID score is heavy javascript work that keeps the browser's main thread busy, which can delay user interactions. We improved FID in the following ways.
+
+### Break up long Javascript tasks
+
+Long tasks are tasks that are 50 milliseconds or longer. Long tasks occupy the browser's main thread and prevent it from responding to user input. We broke up long running tasks into smaller tasks where possible on user request, which helped to reduce the Javascript bloat.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/rfFVGpifiTIaiMIFpTOv.png", alt="A screenshot of an pie chart breaking down CPU time by activity type in the performance panel of Chrome's DevTools. 143 milliseconds was spent scheduling the loading of resources. 4553 milliseconds was spent on JavaScript. 961 milliseconds was spent on rendering work. 191 milliseconds was spent on painting operations. 1488 milliseconds on system tasks, with 3877 milliseconds of idle time. The total timeframe was 11212 milliseconds.", width="800", height="501" %}
+</figure>
+
+### Defer unused javascript
+
+We prioritized page content over third-party scripts such as analytics to keep the page more responsive. However, there are certain limitations on some libraries since they need to be loaded in the document `<head>` in order to accurately track the user journey.
+
+### Reduce Polyfills
+
+We reduced our dependency on certain polyfills and libraries, since browsers provide support for modern APIs, and less users are using legacy browsers, such as Internet Explorer.
+
+### Lazy load ads
+
+Lazily loading below-the-fold ads helped cut down main thread blocking time and thereby improved FID.
+
+### Summarizing FID Goals and achievements
+
+From routine experiments, We were able to reduce our FID from 200 ms to standing under 50 ms today.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/bqZP6cbsJhryp5E17dPV.png", alt="A screenshot of FID distributions as shown in the Chrome User Experience Report. 86% of CLS values are 'Good', 10% are 'Fair', and 4% are 'Poor'. The 75th percentile of user experiences on The Economic Times website overall experienced a FID of 44 milliseconds.", width="720", height="190" %}
+</figure>
+
+## Preventing regressions
+
+Economics Times is planning to introduce automated performance checks in production to avoid page performance regressions. They are planning to evaluate Lighthouse-CI to automate lab tests which can prevent regressions on their production branch.
```

Comment:
```suggestion
The Economics Times plans to introduce automated performance checks in production to avoid page performance regressions. They plan to evaluate Lighthouse-CI to automate lab tests, which can prevent regressions on their production branch.
```